src/Microsoft.ML.Featurizers/DateTimeTransformer.cs

```diff
@@ -570,8 +572,7 @@ internal byte[] CreateTransformerSaveData()
                 if (!success)
                     throw new Exception(GetErrorDetailsAndFreeNativeMemory(errorHandle));
 
-                using (var savedDataHandle = new SaveDataSafeHandle(buffer, bufferSize))
-                {
+                using (var savedDataHandle = new SaveDataSafeHandle(buffer, bufferSize)) {
```

Comment:
I don't think this is the C# convention. Can you please use the previous format of braces on a separate line?

Reply:
Good catch. Fixed that here and everywhere else it occurs.